spec/features/campaign_requests_spec.rb

```diff
@@ -59,7 +59,7 @@
 pensions, campaign, newcampaign
 
 [Site build budget / costs (and overall campaign cost, if applicable)]
-1200
+£1200 and tuppence
```

Comment:
```suggestion
£1200 and two pence
```
Not a blocker but it'd be good to use language that doesn't create barriers to non-natives.